src/CoreWCF.BuildTools/src/DiagnosticDescriptors.cs

```diff
@@ -15,5 +15,41 @@ internal static class DiagnosticDescriptors
                                                                               isEnabledByDefault: true);
 
         internal static Diagnostic RaiseParentClassShouldBePartialError(string parentClassName, string methodName, Location location) => Diagnostic.Create(ParentClassShouldBePartialError, location, parentClassName, methodName);
+
+        internal static readonly DiagnosticDescriptor AllowAnonymousAttributeIsNotSupported = new DiagnosticDescriptor(id: "COREWCF_0200",
+            title: "[AllowAnonymous] attribute is not supported",
+            messageFormat: "[AllowAnonymous] attribute is not supported",
+            category: nameof(AuthorizationAttributesAnalyzer),
+            DiagnosticSeverity.Warning,
+            isEnabledByDefault: true);
+
+        internal static Diagnostic AllowAnonymousAttributeIsNotSupportedWarning(Location location) => Diagnostic.Create(AllowAnonymousAttributeIsNotSupported, location);
+
+        internal static readonly DiagnosticDescriptor AuthorizeAttributeIsNotSupportedOnClass = new DiagnosticDescriptor(id: "COREWCF_0201",
+            title: "[Authorize] attribute is not supported",
+            messageFormat: "[Authorize] attribute is not supported on class '{0}'",
```

Comment:
Should we clarify in the message that it's not supported on classes? This message sounds more generally like it's not supported by CoreWCF at all.

Reply:
done in https://github.com/CoreWCF/CoreWCF/pull/850/commits/dc5c90ab00ffd1908065e943ee07f5530ee5b464